src/parser/jobs/mnk/Fists.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash'
+import React, {Fragment} from 'react'
+
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+
+import Module from 'parser/core/Module'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+const STANCES = [
+	STATUSES.FISTS_OF_EARTH.id,
+	STATUSES.FISTS_OF_FIRE.id,
+	STATUSES.FISTS_OF_WIND.id,
+]
+
+const STANCELESS_SEVERITY = {
+	0: SEVERITY.MEDIUM,
+	12: SEVERITY.MAJOR,
+}
+
+export default class Fists extends Module {
+	static handler = 'fists'
+	static dependencies = [
+		'suggestions',
+	]
+
+	_stanceApplied = []
+	_stanceDropped = []
+	_stanceless = 0
+
+	constructor(...args) {
+		super(...args)
+		this.addHook('applybuff', {to: 'player', abilityId: STANCES}, this._onGain)
+		this.addHook('removebuff', {to: 'player', abilityId: STANCES}, this._onRemove)
+		this.addHook('complete', this._onComplete)
+	}
+
+	_onGain(event) {
+		if (this._stanceDropped.length > 0) {
+			const lostTime = event.timestamp - _.last(this._stanceDropped).timestamp
+			if (lostTime > 0) {
+				this._stanceless += lostTime
+			}
+		}
+
+		this._stanceApplied.push(event)
+		return
+	}
+
+	_onRemove(event) {
+		this._stanceDropped.push(event)
+	}
+
+	_onComplete() {
+		if (this._stanceless > 0) {
```

Comment:
For ref, you can remove the guard  if you're using `TieredSuggestion`. Values under the smallest severity in your tier list will automatically fall back to `SEVEIRY.IGNORE`, which just hides it.